tslearn/barycenters/dba.py

```diff
@@ -514,6 +534,7 @@ def dtw_barycenter_averaging(X, barycenter_size=None, init_barycenter=None,
 def dtw_barycenter_averaging_one_init(X, barycenter_size=None,
                                       init_barycenter=None,
                                       max_iter=30, tol=1e-5, weights=None,
+									  n_treads=15,
```

Comment:
We use `n_jobs` everywhere as a parameter name, and `None` as its default, we should stick to the same convention here, I think.